onnxruntime/core/providers/cuda/tensor/slice.cc

```diff
@@ -63,46 +63,43 @@ REGISTER_V11_TYPED_SLICE(int32_t)
 REGISTER_V11_TYPED_SLICE(int64_t)
 REGISTER_V11_TYPED_SLICE(float)
 
-template <bool dynamic>
-Status Slice<dynamic>::ComputeInternal(OpKernelContext* ctx) const {
-  const Tensor* input_tensor = GetSlicedOrUnslicedTensor(ctx);
+static Status SliceImpCore(const void* input_data, void* output_data,
```

Comment:
Core imp shared by `CallSliceImp()`  (called by `ComputeInternal()`) and `SliceCuda::Imp()` (stand-alone core Slice implementation for other callers) to avoid code duplication